### ISSUE TITLE
Feature Request — Signature Help

### ISSUE THREAD

**Issue by nqthqn** [Krzysztof-Cieslak/vscode-elm#200]
I thought this was really cool! Does vscode have support for something like this?

![function](https://user-images.githubusercontent.com/857356/38226071-96d21f82-36ac-11e8-9d21-9a868dc28484.gif)

**Comment by Krzysztof-Cieslak** [comment]
Yes, VSCode has an API for such helpers - https://code.visualstudio.com/docs/extensionAPI/vscode-api#SignatureHelpProvider

**Comment by tylerthehaas** [comment]
I would like to start work on this issue. Any objections?
